# This file is distributed under the same license as the Django package.
#
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-03-16 04:06+0000\n"
"PO-Revision-Date: 2010-05-13 15:35+0200\n"
"Last-Translator: Django team\n"
"Language-Team: English <en@li.org>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:49
msgid "Important dates"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:135
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:145
msgid "Password changed successfully."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/admin.py:165
#, python-format
msgid "Change password: %s"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/apps.py:14
msgid "Authentication and Authorization"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/base_user.py:48
msgid "password"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/base_user.py:49
msgid "last login"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:30
msgid "No password set."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:35
msgid "Invalid password format or unknown hashing algorithm."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:70
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:307
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:379
msgid "The two password fields didn't match."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:73
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:126
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:163
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:383
msgid "Password"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:79
msgid "Password confirmation"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:82
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:392
msgid "Enter the same password as before, for verification."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:128
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:170
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:173
msgid "This account is inactive."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:234
msgid "Email"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:310
msgid "New password"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:316
msgid "New password confirmation"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:351
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:354
msgid "Old password"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/forms.py:389
msgid "Password (again)"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:263
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:337
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:434
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:496
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:527
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:563
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:599
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:637
msgid "algorithm"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:264
msgid "iterations"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:265
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:343
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:436
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:497
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:528
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:638
msgid "salt"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:266
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:344
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:498
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:529
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:564
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:600
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:639
msgid "hash"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:338
msgid "variety"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:339
msgid "version"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:340
msgid "memory cost"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:341
msgid "time cost"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:342
msgid "parallelism"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:435
msgid "work factor"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/hashers.py:437
msgid "checksum"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:56
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:111
msgid "name"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:60
msgid "content type"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:62
msgid "codename"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:66
msgid "permission"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:67
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:114
msgid "permissions"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:121
msgid "group"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:122
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:218
msgid "groups"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:209
msgid "superuser status"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:212
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:221
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:229
msgid "user permissions"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:231
msgid "Specific permissions for this user."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:298
msgid "username"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:301
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:304
msgid "A user with that username already exists."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:307
msgid "first name"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:308
msgid "last name"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:309
msgid "email address"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:311
msgid "staff status"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:313
msgid "Designates whether the user can log into this admin site."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:316
msgid "active"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:319
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:323
msgid "date joined"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:332
msgid "user"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/models.py:333
msgid "users"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:102
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] ""
msgstr[1] ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:112
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] ""
msgstr[1] ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:151
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:157
msgid "Your password can't be too similar to your other personal information."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:185
msgid "This password is too common."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:190
msgid "Your password can't be a commonly used password."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:200
msgid "This password is entirely numeric."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/password_validation.py:205
msgid "Your password can't be entirely numeric."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/validators.py:12
msgid ""
"Enter a valid username. This value may contain only English letters, "
"numbers, and @/./+/-/_ characters."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:180
msgid "Logged out"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:282
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:401
msgid "Password reset"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:297
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:429
msgid "Password reset sent"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:335
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:438
msgid "Enter new password"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:346
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:498
msgid "Password reset unsuccessful"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:366
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:506
msgid "Password reset complete"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:541
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:569
msgid "Password change"
msgstr ""

#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:557
#: ll_env/lib/python3.6/site-packages/django/contrib/auth/views.py:592
msgid "Password change successful"
msgstr ""
